content/get_start/demo/demo1.md

```diff
@@ -1,96 +1,144 @@
-demo_path = "#{File.dirname(Dir.pwd)}/source"
-proposed_geo_model = "/#{demo_folder}/#{proposed_model}"
-user_data_path = "#{demo_path}/#{demo_folder}/user_data"
-```
+2. **Load the .osm file:** Go to the File menu and select Open. Navigate to the downloaded folder `demo 1` and select `geometry_demo_model.osm` file. The project inputs like weather file, climate zone and location will load automatically (as shown in the image). If any parameter is missing, add it manually, for example, if the weather file is missing, select the .epw file provided in the folder or any other before moving forward.
+![demo1_loadosmfile image](/BEM-for-PRM/get_start/demo/images/demo1_loadosmfile.PNG?width=1400px&align=right&classes=border,alignLeft)
 
-There are four directories need attentions.
+3. **Add the measure:** Go to the **_Measures_** tab in the left menu. Under Whole Building > Space Types, *_Create ASHRAE 90.1 2019 PRM Model_* measure should be available. If you don't see it, revisit the steps in [Use with OS App](http://localhost:1313/BEM-for-PRM/get_start/os_app/) section. Add the measure.
 
-- `baseline_dir` is the path used by PRM method to save all output files.
-- `demo_path` is the parent path for demo folder
-- `proposed_geo_model` is the relative path of proposed model. The path shall be relative to the `demo_path`
-- `user_data_path` is the path to user data folder, which contains all the user data `.csv`.
+4. **Setting Project Parameters:** Click on the measure after adding it, a few project parameters will show up in the **Edit** menu on the right. Under **Inputs**, select the following: 
+    - Default Building Area Type for Window To Wall Ratio : **_Office 5,000 to 50,000 sq ft_**
+    - Default Building Area Type for Service Water Heating : **_Office_**
+    - Default Building Type for HVAC : **_other nonresidential_**
+    - Climate Zone : **_ASHRAE 169-2013-4A_**
+    - Exempt from Rotations : **_FALSE_** (You can select TRUE for your project if the building has rating authority approval that orientation is dictated by site considerations.)
+    - Exempt from Unmet Load Hours Check : **_TRUE_** (TRUE has been selected here only for demo purposes, otherwise it's always recommended to select FALSE until and unless the rating authority has approved allowing the models to exceed the specified.)
+    - Use User Data : **_FALSE_** (FALSE selected since we are not using any user data for this demo.) 
 
-{{< line_break >}}
+5. **Run Simulation:** Go to the **_Run Simulation_** tab and run the  simulation. Once it's completed, go to the *_demo1_* folder in the system. A new folder `geometry_demo_model` would be created containing subfolders for the runs as shown in [Run the measure] section. The `generated_files` folder would contain the output .osm files, reports that should reflect the Output shown [here](http://localhost:1313/BEM-for-PRM/get_start/demo/demo1/#demo-1-results). 
```

Comment:
You will need to update this link - use full link will fail the production deployment.
Use relative path: `/BEM-for-PRM/get_start/demo/demo1/#demo-1-results`